Exec/test_problems/double_bubble/inputs_2d_C

```diff
@@ -93,36 +93,33 @@ maestro.small_temp = 1.e-5
 # BURNING parameters
 maestro.do_burning = false
 
-&probin
-
-  ! override the default values of the probin namelist values here
-  pert_factor = 8.1e-3
-  pres_base = 1.65d6
-  dens_base = 1.65d-3
-  y_pert_center = 0.7d0
-  pert_width = 0.025d0
-  do_isentropic = T
-  single = F
-
-  !extern
-
-  ! Note that some of the parameters in this
-  ! namelist are specific to the default EOS,
-  ! network, and/or integrator used in the
-  ! makefile. If you try a different set of
-  ! microphysics routines be sure to check that
-  ! the parameters in here are consistent, as
-  ! Fortran does not like seeing unknown variables
-  ! in the namelist.
-
-  const_conductivity = 1.0d0
-  species_a_name = "A"
-  species_a_gamma = 1.666d0
-
-  species_b_name = "B"
-  species_b_gamma = 1.4d0
-
-  species_c_name = "C"
-  species_c_gamma = 1.666d0
-
-/
+# override the default values of the probin namelist values here
+dens_base = 1.65e-3
+problem.pert_factor = 8.1e-3
+dens_base = 1.65e-3
+problem.pres_base = 1.65e6
+dens_base = 1.65e-3
+problem.dens_base = 1.65e-3
+problem.y_pert_center = 0.7e0
+problem.pert_width = 0.025e0
+problem.do_isentropic = 1
+problem.single = 0
+
+# Note that some of the parameters in this
+# namelist are specific to the default EOS,
+# network, and/or integrator used in the
+# makefile. If you try a different set of
+# microphysics routines be sure to check that
+# the parameters in here are consistent, as
+# Fortran does not like seeing unknown variables
+# in the namelist.
+
+eos.const_conductivity = 1.0d0
```

Comment:
this should be `conductivity.const_conductivity`, also get rid of the `d` -- C++ doesn't like that.